services/httpd/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	WriteTracing            bool              `toml:"write-tracing"`
 	FluxEnabled             bool              `toml:"flux-enabled"`
 	FluxLogEnabled          bool              `toml:"flux-log-enabled"`
+	FluxTestingEnabled      bool              `toml:"-"`
```

Comment:
If we just left this on always, what would be the effect? Just that users would be able to write flux tests for their own custom queries?

Reply:
Jonathan says it's harmless to always have on, so I'll refactor to simplify